kuksa_databroker/build-all-targets-cli.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
What is our view for *.sh files. Shall we have copyright+license header? Currently not required by our SPDX checker

Reply:
Hmm maybe I'll add at least a SPDX-header to make scanners happy. I am always a bit torn, normal these things are not very magic to warrant "protection", but likely putting license explicitly is better